src/js/PhotoSphereViewer.public.js

```diff
@@ -487,3 +487,45 @@ PhotoSphereViewer.prototype.startKeyboardControl = function() {
 PhotoSphereViewer.prototype.stopKeyboardControl = function() {
   window.removeEventListener('keydown', this);
 };
+
+/**
+ * Preload a panorama image and save it into internal cache.
+ * @param {String} pano - the file path
+ * @param {Function} callback - Progress callback, will receive the percentage as argument.
+ */
+PhotoSphereViewer.prototype.preloadPano = function(pano, callback) {
```

Comment:
this callback a very little interest and is incoherent with the whole promise based API.
please remove it
+ missing the @returns annotation